content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst

```diff
@@ -126,79 +126,100 @@ Registration
-   - `Peppol Endpoint - OpenPeppol eDEC Code Lists <https://docs.peppol.eu/edelivery/codelists/>`_
-     (open the "Participant Identifier Schemes" as HTML page)
+.. tip::
+   - You can always register as a sender first and register to receive documents later.
+   - You can specify when registering if you would also like to receive documents.
 
-If you are migrating from another access point, insert the :guilabel:`Migration key` from the
-previous provider.
+.. image:: electronic_invoicing/peppol-registration-wizard.png
+   :alt: Peppol registration form
 
-.. image:: electronic_invoicing/peppol-settings.png
-   :alt: Configuration for peppol
+Fill in the following information:
 
-Finally, click :guilabel:`Validate registration`. If you want to explore or demo Peppol, you can
-choose to register in :guilabel:`Demo` mode. Otherwise, select :guilabel:`Live`.
+- Check the receiver box if you want to register on Odoo SMP. If you are migrating from another
+  service provider, insert the :guilabel:`Migration key` from the previous provider in a field that
+  will become visible after you tick the checkbox.
+- :guilabel:`E-Address Scheme`: the Peppol Electronic Address Scheme and usually depends on your
+  company's country. Odoo often prefills this with the most commonly used EAS code in your country.
+  For example, the preferred EAS code for most companies in Belgium is `0208`.
+- :guilabel:`Endpoint`: this is usually a Company Registry number or a VAT number.
+- :guilabel:`Phone`: phone number including the country code (e.g., `+32` in Belgium).
+- :guilabel:`Email`: this the email Odoo can contact you to regarding your Peppol registration.
 
-   .. image:: electronic_invoicing/peppol-demo-mode.png
-      :alt: Peppol demo mode selection
+If you want to explore or demo Peppol, you can choose to register in :guilabel:`Demo` mode.
+Otherwise, select :guilabel:`Live`.
 
-.. note::
-   When testing Peppol, the system parameter `account_peppol.edi.mode` can be changed to `test`.
-   Then, a radio button appears with the option to register on the test server.
+.. tip::
+   - Selecting :guilabel:`Demo` simulates everything in Odoo. There is no sending, receiving, or
+     partner verification.
+   - For **advanced users only**, it is possible to run tests on Peppol's test network. The server
+     allows to register on Peppol, send, and receive test invoices to and from other
+     participants. To do so, enable the :doc:`../../../general/developer_mode`, open the
+     **Settings** app, go to :menuselection:`Technical --> System Parameters`, and search for
+     `account_peppol.edi.mode`. Open the parameter and change the :guilabel:`Value` to `test`. Go
+     back to the Peppol setup menu in the **Settings** app. The option :guilabel:`Test` is now
+     available.
 
    .. image:: electronic_invoicing/peppol-system-parameter.png
       :alt: Peppol test mode parameter
```

Comment:
I would remove this screenshot (and the corresponding image file). We're trying to limit screenshots as much as possible and I don't think this one adds much value to the content here.

Reply:
I have just linked that part to a customer yesterday where they were confused about system parameters... so I think it could be useful?